Formula/aribb24.rb

```diff
@@ -0,0 +1,36 @@
+class Aribb24 < Formula
+  desc "Library for ARIB STD-B24, decoding JIS 8 bit characters and parsing MPEG-TS"
+  homepage "https://github.com/nkoriyama/aribb24"
+  url "https://github.com/nkoriyama/aribb24/archive/refs/tags/v1.0.3.tar.gz"
```

Comment:
maybe we should use `videolan` one instead, it is 1.0.4 release there.

Reply:
The 1.0.4 there isn't a tagged release; I've asked JEEB to make one, and if that happens, I'll update this to point at it. Unless you're looking at a different repo than I am?

Reply:
Yeah, I was looking at a different repo. (this one, https://code.videolan.org/jeeb/aribb24)

What is the difference between the github and the gitlab?

Reply:
The one under videolan's gitlab is effectively just JEEB's personal fork at the moment, though it's likely to be brought in as an official videolan project at some point in the future (in which case it would move to `https://code.videolan.org/videolan/aribb24`). JEEB's repo has the version bumped to 1.0.4, but doesn't have a "v1.0.4" tag in git, so I'd have to point at a commit by hash to use it.

Reply:
JEEB's tagged 1.0.4 in that fork, so I've updated this PR to use it. This also resolves the `brew audit --new` failure (…since there's no longer a link to an infrequently-used gitlab repo).

Reply:
@chenrui333 ping on this?